test/uk/gov/hmrc/cdsreimbursementclaimfrontend/journeys/SecuritiesJourneyFormatSpec.scala

```diff
@@ -35,6 +35,25 @@ class SecuritiesJourneyFormatSpec extends AnyWordSpec with JsonFormatTest with M
     PropertyCheckConfiguration(minSuccessful = 100)
 
   "SecuritiesJourney.Answers" should {
+
+    "println sample journey in JSON format" in {
```

Comment:
I am not sure this should be a test, could we put this on a test only page?

Reply:
we cannot unfortunately put it on the main code as it depends on the test dependencies heavily

Reply:
but we can remove it at some point